docs/develop/05_dApp/verifier/04_sendRequest.md

```diff
@@ -0,0 +1,12 @@
+---
+id: send
+title: Send Request
+---
+
+import CodeBlock from '@theme/CodeBlock';
+import SendRequest from '!!raw-loader!@site/code_examples/workshop/verifier/sendRequest.ts';
+
+Credential Request should be sent from the frontend to extension. Extension sends back the requested credential. That credential is then sent for verification to the backend.
```

Comment:
is this about how to send a request in general?

Reply:
Yes, its more about sending from frontend.

Reply:
So the frontend might send a message, but the backend does too? How does the frontend send a message. They are encrypted, right?